packages/dd-trace/test/plugins/helpers.js

```diff
@@ -2,11 +2,14 @@
 
 const { AssertionError } = require('assert')
 
-function expectSomeSpan (agent, expected) {
+function expectSomeSpan (agent, expected, logSpans = false) {
```

Comment:
Isn't the span always logged if the test doesn't pass?

Reply:
If the test doesn't pass, *one* error is given. This allows me to see *all* the spans, in case the error message was not enough. I'm not sure it makes sense to keep this on by default, since it starts cluttering the test output.

Reply:
I think it could be on by default if it's part of the error message and not a separate `console.log`. Is that something that could be done?

Reply:
So you want the error message to contain every span received in the test? That seems awfully verbose. But also helpful, so I'll give that a try!

Reply:
To be honest I'm not sure, as you said it might be too verbose. But at the same time you would be guaranteed to have the correct span show up since all spans would show up.

I'm just not a fan of an option that is only used by making a temporary code change. Especially since adding a `console.log` directly in the test is not really more effort than changing the call to `expectSomeSpan`.